api/registration/models.py

```diff
@@ -32,13 +33,25 @@ class User(AbstractUser, PermissionsMixin):
     # Identify Information
     id = models.BigAutoField(primary_key=True)
     email = models.EmailField("Email Address", unique=True, blank=False, null=False)
+    
+    USERNAME_FIELD = "email"
+    REQUIRED_FIELDS = []
 
+    objects = UserManager()
+
+    @property
+    def full_name(self):
+        return '%s %s' % (self.first_name, self.last_name)
```

Comment:
This custom property was made to create a full name field from the first name field and the last name field.
Please move it to the `UserInformation` model.